packages/server/src/types.ts

```diff
@@ -27,4 +27,6 @@ export type NextAppOptions = Parameters<typeof next>[0];
 export type LlamaIndexServerOptions = NextAppOptions & {
   workflow: WorkflowFactory;
   starterQuestions?: string[];
+  appTitle?: string;
+  useLlamaCloud?: boolean;
```

Comment:
1. @leehuwuj the llamacloud parameter should be in sync with the python version
2. seems like the idea is that if this is true then these env variables must be defined:
LLAMA_CLOUD_API_KEY
LLAMA_CLOUD_INDEX_NAME
LLAMA_CLOUD_PROJECT_NAME

how about passing them instead as object:
```
llamaCloud: {
apiKey,
index,
project
}
```
if the object is undefined/null (or `llamaCloud.apiKey` is undefined/null) we don't use llamacloud

then create-llama is generating this object with:
```
llamaCloud: {
apiKey: process.env.LLAMA_CLOUD_API_KEY
index: process.env.LLAMA_CLOUD_INDEX_NAME
project: process.env.LLAMA_CLOUD_PROJECT_NAME
}
```
AFAIK we can't modify the LlamaIndexServer parameters easily